ios/README.md

```diff
@@ -0,0 +1,72 @@
+
+## Prerequisites
+- iOS device (iOS version 11.0 or higher)
```

Comment:
Did we test it with iOS 11 ?

Reply:
config build limit for iOS 11 or higher. We can not test on the all os version we support.

Reply:
ok